src/web/components/Subtitle/Subtitle.test.js

```diff
@@ -6,7 +6,7 @@ describe('Subtitle', () => {
   let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(<Subtitle>Hello World</Subtitle>);
+    wrapper = shallow(<Subtitle>Hello World</Subtitle>).shallow();
```

Comment:
Why to shallow the shallowed result ?

Reply:
I was using shallow to render the component in the HOC. But after our discussion, I ran the tests without the hoc.
Thank you